Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:20.04
+RUN apt-get update && apt-get upgrade --yes
+RUN apt-get install python3-pip python3.8 --yes
+RUN pip3 install --upgrade eth-ape
+RUN ape plugins add ens --yes
+RUN ape plugins add etherscan --yes
+RUN ape plugins add infura --yes
+RUN ape plugins add solidity --yes
+RUN ape plugins add vyper --yes
```

Comment:
A lot of CLI Dockerfiles I see use ENTRYPOINT, so it is like running ape when you run the container, which means if you don't give it commands, it just prints the help and exits... I think you mentioned you looked into this pattern, but just thought I'd say that it is common from what I have seen. How much have we considered this approach?

Reply:
Here is GETH example: https://github.com/islishude/geth-docker/blob/master/Dockerfile

Reply:
Notice it builds uses alpine, uses `geth` command as the entrypoint, and it builds from source!

Reply:
It would be what ever command the user types in!

They would run the container with the same args that they run `ape` with. So if they run the container with `console` as the argument, it would launch a console, etc.

Reply:
Like this `docker run apeworx/ape console

Reply:
> It would be what ever command the user types in!
> They would run the container with the same args that they run `ape` with. So if they run the container with `console` as the argument, it would launch a console, etc.

yes then the problem here then becomes that they can't do other things if they need to with the cli within that image, right?